docs/src/content/docs/fr/resources/community-content.mdx

```diff
@@ -127,5 +127,12 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
 			description:
 				"Créez un nouveau site Starlight prêt à l'emploi en environ 5 minutes",
 		},
+		{
+			href: 'https://www.youtube.com/watch?v=12o7WxjAxjM',
+			title:
+				'Inclure une documentation Starlight dans un projet Next.js avec des proxys',
```

Comment:
Decided to keep "proxy" as-is as in this context, we do not have (afaik) any proper translation and [FranceTerm](https://www.culture.fr/franceterme) does not have one either.